crestonesoftware/boombastic-froggy
# Updates Community GET route to include events and updates event-details.handlebars

❗ Please **DO NOT** open a Pull Request without creating an Issue first. Failure to do so will result in the rejection of the pull request.

## Purpose

Updates the event-details page and renders them on the screen. Adds Event information in the GET Community route on home-routes.js

Fixes #175 

## Changes


## Steps to Reproduce or Test

Navigate to the Chacala community page and click Events to see them populate below.

## Optional GIF

Self-explanatory.
